projects/metis/src/app/dataset/reportsimple/reportsimple.component.html

```diff
@@ -1,102 +1,107 @@
-<div class="modal-wrapper" *ngIf="isVisible">
-  <div class="modal modal-big">
-    <div class="head">
-      Report
-      <span *ngIf="reportRequest.pluginType; let plugin">({{ plugin | renameWorkflow }})</span
-      ><a class="btn-close" (click)="close()"></a>
-      <a
-        class="btn-right"
-        (click)="copyReport()"
-        *ngIf="reportRequest.errors || reportRequest.message"
-        >Copy</a
-      >
+<ng-template #tmpBtnCopy>
```

Comment:
The name of this template `tmpBtnCopy` is not clear. Is it meant to be `Template Button Copy`?
Could you change it to a name that is easier to read/understand?

Reply:
The conventions `tmp` and `btn` are used widely in the project, so this is consistent.

Reply:
In my opinion maybe button copy needs to have a more representative/descriptive name

Reply:
I've renamed it to `tmpBtnCopyReport`.  I think that encapsulates all that a name can.